spec/brcobranca/retorno/cnab400/itau_spec.rb

```diff
@@ -0,0 +1,55 @@
+# -*- encoding: utf-8 -*-
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.